examples/federated_learning/surface_defect_detection/aggregation_worker/aggregate.py

```diff
@@ -26,8 +26,8 @@ def run_server():
     participants_count = int(Context.get_parameters(
         "participants_count", 1
     ))
-    agg_ip = Context.get_parameters("AGG_IP", "0.0.0.0")
-    agg_port = int(Context.get_parameters("AGG_PORT", "7363"))
+    agg_ip = Context.get_parameters("AGG_BIND_IP", "0.0.0.0")
```

Comment:
There are two places reading these two environment variables `AGG_BIND_IP`/`AGG_BIND_PORT`, and only one place takes effect. I suggest to keep the one in `lib/sedna/service/server/aggregation.py`, and remove this example place.

Reply:
Actually the example should not set these values.

Reply:
check